recipeList/recipeJS.js

```diff
@@ -0,0 +1,43 @@
+function addIngredient() {
+    var ulUnorder = document.getElementById("ingredients"); //gets the list
+    var liElement = document.createElement("li"); //creates new element
+    var ingredient = document.getElementById("addIngredient").value;
```

Comment:
In general, its recommended to use let instead of var. Here's a little comparison between the two: https://stackoverflow.com/a/11444416